test/e2e/antreapolicy_test.go

```diff
@@ -821,6 +883,81 @@ func testACNPDropEgress(t *testing.T) {
 	executeTests(t, testCase)
 }
 
+// testACNPDropIngressUDP tests that a ACNP is able to drop ingress traffic from pods labelled A to namespace Z in UDP.
+func testACNPDropIngressUDP(t *testing.T) {
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-drop-a-from-z-ingress-udp").
+		SetPriority(1.0).
+		SetAppliedToGroup([]ACNPAppliedToSpec{{PodSelector: map[string]string{"pod": "a"}}})
+	builder.AddIngress(v1.ProtocolUDP, &p5000, nil, nil, nil, nil, map[string]string{"ns": "z"},
+		nil, nil, nil, secv1alpha1.RuleActionDrop, "", "")
+
+	reachability := NewReachability(allPods, Connected)
+	reachability.Expect(Pod("z/a"), Pod("x/a"), Dropped)
+	reachability.Expect(Pod("z/b"), Pod("x/a"), Dropped)
+	reachability.Expect(Pod("z/c"), Pod("x/a"), Dropped)
+	reachability.Expect(Pod("z/a"), Pod("y/a"), Dropped)
+	reachability.Expect(Pod("z/b"), Pod("y/a"), Dropped)
+	reachability.Expect(Pod("z/c"), Pod("y/a"), Dropped)
+	reachability.Expect(Pod("z/b"), Pod("z/a"), Dropped)
+	reachability.Expect(Pod("z/c"), Pod("z/a"), Dropped)
+
+	testStep := []*TestStep{
+		{
+			"Port 5000",
+			reachability,
+			[]metav1.Object{builder.Get()},
+			nil,
+			[]int32{5000},
+			v1.ProtocolUDP,
+			0,
+			nil,
+		},
+	}
+	testCase := []*TestCase{
+		{"ACNP Drop UDP ingress from NS:z to All Pod:a", testStep},
+	}
+	executeTests(t, testCase)
+}
+
+// testACNPDropIngressSCTP tests that a ACNP is able to drop ingress traffic from pods labelled A to namespace Z in SCTP.
+func testACNPDropIngressSCTP(t *testing.T) {
+	t.Skipf("OVS userspace conntrack does not have the SCTP support for now.")
```

Comment:
we have a better helper function to skip tests specifically on Kind, please search through the existing test code

this line will skip the test on Jenkins (VMC testbeds) as well, which is definitely not what we want

Reply:
Got it.